FF8.Core/Sources/MusicName.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+
+// ReSharper disable StringLiteralTypo
+
+namespace FF8.Core
+{
+    public static class MusicName
+    {
+        public static String Get(MusicId id)
+        {
+            if (_dic.TryGetValue(id, out var name))
+                return name;
+
+            return $"Unknown music: {id}";
+        }
+
+        private static readonly Dictionary<MusicId, String> _dic = new Dictionary<MusicId, String>()
+        {
+            {MusicId.Loser, "The Loser"},
```

Comment:
I see it's only for debugging purpouses- though it should be eventually parsed from .sgt files directly.

Reply:
oh the sgt has titles in it? I have a dictionary like this in memory.

Reply:
Oh, I'm sorry. I made a mistake- they do indeed have a name... but it's the same name as filename. 
Example: 026s-garden.sgt:0xD4 Unicode string: ```026-garden```